content/post/career-ladders-part-3.md

```diff
@@ -0,0 +1,57 @@
+- About 6 months later, you’ll check back in and see how the plan has gone. Keep a running document, with descriptions or links to work demonstrating various competencies. Hopefully, the assessment looks a little bit different as they’ve worked on a growth area, and you can determine a new area of focus. And if the person is nearing the next level across competencies, you can talk about when that person should be nominated or nominate themselves for a promotion.
+
+These structured conversations help to break down growth plans into achievable scope and align the needs of the organization (in the form of upcoming projects) with the personal desires of your team. They also create a running record of growth that can be used in formal performance reviews and promotion nominations.
+
+### Managing Upward
+Before promotion time approaches, make sure you’re communicating with your manager about where your team members stand and when you’d like to nominate them (or when they’ll likely be nominating themselves, if that’s how your org works). Understand the promotion process, likelihood of promo approval, any limits on the number of promotions (e.g. no more than 20% of employees should be promoted per year), and how you can help your team be ready. Use that information to set expectations appropriately for your team, and find avenues for growth even if it they aren't likely to get a promotion this cycle.
+
+### Promotion Announcements
+Announcing promotions are one of the most fun parts of managing a team. First, telling the person who is being promoted, then telling the entire company - it's chock full of warm fuzzies. But promotion announcements are also a powerful opportunity to reinforce the principles that underpin your career ladder to both the person being promoted and everyone else. What work did they do that showed they had made that step function from one role to another? If there are undersung skills they demonstrated - unblocking and mentoring teammates, or focusing on automation to empower people to focus on higher-value work - now is the to make sure everyone knows that these things really do help you reach the next level.
+
+### Hiring
+Talk to candidates about the career ladder, and how you use it - the fact that the company takes a clear and structured approach to coaching and development is a significant selling point. Within the hiring process, use your ladder to align candidates with levels. Create interview questions designed to tease out where someone is in relation to your levels, and hire them accordingly. Make it clear when you give them an offer what level they are being offered, and talk about level before they accept. The level you're offering may not align with their current level, or where they think they belong, so make sure they understand why that level, and what it will take to get to the next level.
+
+## What If You Don't Have a Career Ladder?
```

Comment:
Might want to clarify a bit more explicitly that you're now talking from the perspective of an individual trying to get  promotion. This a big switch in perspective and may be helpful to more explicitly signal.